source/manual/get-started.html.md

```diff
@@ -105,6 +108,41 @@ Some processes are dependent on your local gov.uk setup being in `~/govuk`:
     mac$ cd ~/govuk
     mac$ git clone git@github.com:alphagov/govuk-puppet.git
 
+### 3.1 Ensure your Ruby setup works
+
+You'll want to install the [specific version of Ruby in govuk-puppet][https://github.com/alphagov/govuk-puppet/blob/master/.ruby-version#L1] (currently `rbenv install 1.9.3-p550`), as the default system Ruby is too locked-down.
```

Comment:
Probably simpler not to mention the current version as this will go out of date very quickly.

Reply:
Yep, it's a violation of DRY and doesn't feel like a good thing to hard-code here. On the other hand, `.ruby-version` hasn't been updated for 3 years... 🤔 and I could see people misinterpreting the docs and running `rbenv install 1.9.3` (without the 'patch' bit) and finding it doesn't work!

Reply:
Again, sorry it was late in the day and I hadn't grokked that this is for puppet!